app/uk/gov/hmrc/fhregistrationfrontend/views/main_template.scala.html

```diff
@@ -53,7 +60,7 @@
     bodyClasses = bodyClasses,
     sidebar = sidebar,
     contentHeader = contentHeader,
-    mainContent = layouts.article(mainContent),
+      mainContent = layouts.article(mainContent),
```

Comment:
can we remove the layouts import and use the viewHelpers instead. If you clean compile you'll notice if you have warnings about using DI instead of imports.

Reply:
Also on line 43 layouts is still used